README.md

```diff
@@ -1320,6 +1379,15 @@ __AWESOME__ apps counter: __150__ 🎉
 - [x] [GitHub](https://github.com/Dimowner/AudioRecorder)
 - [ ] Official page
 
+## VPN
+### Bitmask
```

Comment:
Is it actually good compared to for example ProtonVPN?

Reply:
read the docs

Reply:
https://bitmask.net/en/features

Security features

    Allows anonymous usage
    LEAP providers can enable the VPN usage using an anonymous certificate as verification method. No Passwords needed! 

I don't know much, you can read their documentation https://leap.se/

Reply:
Seems legit, but still I'm not convinced.